sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -2270,7 +2270,7 @@ case class ElementAt(
               case Some(value) =>
                 val defaultValueEval = value.genCode(ctx)
                 s"""
-                  ${defaultValueEval.code};
+                  ${defaultValueEval.code}
```

Comment:
is this a bug fix?

Reply:
No. Please see: https://github.com/apache/spark/pull/37589#pullrequestreview-1081493528